index.css

```diff
@@ -0,0 +1,114 @@
+
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+header {
+  background-color: #3c3a39;
+  width: 100vw;
+  height: 24.5vw;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+}
+
+header * {
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  margin: auto;
+}
+
+header h1 {
+  display: inline;
+  color: white;
+  margin: auto;
+}
+
+#time {
+  display: inline;
+  color: white;
+  margin: auto;
+}
+
+header nav {
+  align-self: center;
+  justify-self: right;
+  justify-content: flex-end;
+  align-items: flex-end;
+  display: inline;
+  position: relative;
+}
+
+#icons {
+  align-self: center;
+  justify-self: right;
+  justify-content: flex-end;
+  align-items: flex-end;
+  display: inline;
+  position: relative;
+  margin-left: 250px;
+}
+
+nav img {
+  margin-left: 180px;
+}
+
+.nav-links {
+  position: absolute;
+}
+
+main {
+  margin-top: 24.5vw;
+  width: 100vw;
+}
+
+.hero {
+  background-color: #1c1a19;
+  width: 100%;
+  height: 157.33vw;
```

Comment:
- [ ] You should try to avoid hard-coding dimensions. It is hard to maintain. So kindly remove all the unnecessary `vw` and use `px` or `%` instead.

Reply:
Okay

Reply:
I calculated the percentages based on the figma design and referenced them to the viewport width of the device. My goal was to make sure that regardless of the width of the device, the dimensions of both the header and hero sections will adjust accordingly with the proportionate dimensions. Do you still think I should change them?

Reply:
@jasonfelice what do you think?